alarmpresenter/main.qml

```diff
@@ -124,18 +125,24 @@ Application {
 
     IconButton {
         id: alarmSnooze
-        iconName: "ios-sleep-circle-outline"
+        iconName: visible && alarmDialog.type === Alarm.Countdown ? "ios-refresh-circle-outline" : "ios-sleep-circle-outline"
```

Comment:
Does it mean that the snooze button appears for calendar events too ? Does it make sense ?

Reply:
Yes, I think it makes sense to have a snooze button for calendar entries. For instance, if I am running late for an appointment, it may be convenient to have the alarm happen again to remind me of some detail such as a room number or the name of the person I am supposed to meet as I arrive at the location.

Reply:
Uh, it totally makes sense now that I read it again... ironically I must have read that in between two snooze sessions :)

Thanks!